util/fhir.ts

```diff
@@ -37,6 +39,14 @@ export function createPatientResourceString(birthDate: string): string {
   return JSON.stringify(pt, null, 2);
 }
 
+export function getFhirResourceString(resource: fhir4.Resource) {
+  const primaryCodePath = parsedPrimaryCodePaths[resource.resourceType].primaryCodePath;
```

Comment:
If the resource doesn't have a primaryCodePath then this throws an error. Check to make sure it has one and fallback to returning something else. Also discussed below.